# Lidar range value from 80째 - 101째 -> Caution median index -> 0째 => ix = [101,...91,90,89,88,...,80]
float32[] Lidar_Range_Values_At_90
# Lidar value at 90째
float32   Lidar_Value_At_90
# mean of range
float32   Mean